modules/schematics/src/reducer/files/__name@dasherize@if-flat__/__name@dasherize__.reducer.ts.template

```diff
@@ -11,11 +13,21 @@ export const initialState: State = {
 
 };
 
-export const reducer = createReducer(
-  initialState,
 <% if(feature) { %>
-  on(<%= classify(name) %>Actions.<%= prefix %><%= classify(name) %>s, state => state),
-<% if(api) { %>  on(<%= classify(name) %>Actions.<%= prefix %><%= classify(name) %>sSuccess, (state, action) => state),
-  on(<%= classify(name) %>Actions.<%= prefix %><%= classify(name) %>sFailure, (state, action) => state),
-<% } %><% } %>
-);
+const feature = createFeature({
+  '<%= camelize(name) %>',
```

Comment:
The current way results in a compilation error because it does not have the correct signature.

```suggestion
  name: '<%= camelize(name) %>',
```

Reply:
Thanks. I also added it in the test-case 👍  Is there an easy way to check what the schematics really generate?

Reply:
I usually log it to the console or create a file with the content.
Maybe using snapshot testing could be helpful here 🤔

Reply:
Hmm can you maybe give me some more steps how to do the logging or creating one? I'm new into developing schematics 🙂 
Yes snapshot testing would be very helpful, but I think this is out of scope of this feature.

Reply:
Oh sorry, I missed this comment.
In the test case we have access to the generated file (e.g. `fileContent` in the test that is updated in this PR). You can a `console.log(fileContent)` or `fs.writeFileSync('print.ts', fileContent)` to see the output.